src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -1969,6 +1969,46 @@ Well{0} entered with 'FIELD' parent group:
         }
     }
 
+    void Schedule::handleWINJCLN(HandlerContext& handlerContext) {
+        for (const auto& record : handlerContext.keyword) {
+            const std::string& wellNamePattern = record.getItem<ParserKeywords::WINJCLN::WELL_NAME>().getTrimmedString(0);
+            const auto well_names = this->wellNames(wellNamePattern, handlerContext);
+            for (const auto& well_name: well_names) {
+                auto well = this->snapshots.back().wells(well_name);
+                well.handleWINJCLN(record, handlerContext.keyword.location());
+                this->snapshots.back().wells.update(std::move(well));
+            }
+        }
+    }
+
+    void Schedule::handleWINJDAM(HandlerContext& handlerContext) {
+        for (const auto& record : handlerContext.keyword) {
+            const std::string& wellNamePattern = record.getItem<ParserKeywords::WINJDAM::WELL_NAME>().getTrimmedString(0);
+            const auto well_names = wellNames(wellNamePattern);
+
+            for (const auto& well_name : well_names) {
+                auto well = this->snapshots.back().wells(well_name);
+                if (well.handleWINJDAM(record, handlerContext.keyword.location())) {
+                    this->snapshots.back().wells.update( std::move(well) );
+                }
+            }
+        }
+    }
+
+    void Schedule::handleWINJFCNC(Schedule::HandlerContext& handlerContext) {
```

Comment:
I didn't notice this before, sorry, but you should not have `Schedule::HandlerContext&` as a parameter type here.  It's better to just write `HandlerContext&` instead like all (most?) other keyword handling functions use.

Reply:
Thanks for noticing. I did not recall what happened there back then half year ago. Will update.